packages/uiweb/src/lib/components/space/SpaceCreationWidget/SCWInviteModal/SCWInviteModal.tsx

```diff
@@ -10,14 +10,20 @@ import { SearchInput } from '../../reusables/SearchInput';
 import { ProfileContainer } from '../../reusables/ProfileContainer';
 import { ThemeContext } from '../../theme/ThemeProvider';
 import { Spinner } from '../../reusables/Spinner';
+import { createIcon } from '../../helpers/blockies';
 
 import CircularProgressSpinner from '../../../loader/loader';
 
 import { useSpaceData } from '../../../../hooks';
 import SettingsIcon from '../../../../icons/settingsBlack.svg';
 import { Image } from '../../../../config';
 
-import { createIcon } from '../../helpers/blockies';
+
+export interface ICustomSearchResult {
```

Comment:
let's make every parameter as optional here and put handle in place of walletAddress